AllReadyApp/Web-App/AllReady/Models/Campaign.cs

```diff
@@ -71,7 +71,7 @@ public class Campaign
 
         public Location Location { get; set; }
 
-        public List<CampaignContact> CampaignContacts { get; set; }
+        public List<CampaignContact> CampaignContacts { get; set; } = new List<CampaignContact>();

```

Comment:
Is this change necessary?  Looking at the code, it looks like EF will create this list on all queries, and the tests create the list for other items.

(If this fixed the issue, let me know, and I'll merge)